bin/seed.js

```diff
@@ -1,23 +1,55 @@
 const {
     db,
-    models: { Category, Product },
+    models: { Category, Product, Country },
 } = require('../server/db')
 
 const init = async () => {
     try {
         await db.sync({ force: true })
-        const [sweet, salty, healthy, frozen] = await Promise.all(
+
+       const [sweet, salty, healthy, frozen] = await Promise.all(
             ['sweet', 'salty', 'healthy', 'frozen'].map((name) => {
                 return Category.create({ name })
             })
         )
+
+        const Australia = await Country.create({
+            name: 'Australia',
+            flag: 'em-flag-au',
+        })
+
+        const China = await Country.create({ name: 'China', flag: 'em-cn' })
+        const French = await Country.create({ name: 'French', flag: 'em-fr' })
+        const India = await Country.create({
+            name: 'India',
+            flag: 'em-flag-in',
+        })
+        const Japan = await Country.create({ name: 'Japan', flag: 'em-jp' })
+        const Singapore = await Country.create({
+            name: 'Singapore',
+            flag: 'em-flag-sg',
+        })
+        const SKorea = await Country.create({
+            name: 'South Korea',
+            flag: 'em-kr',
+        })
+        const Spain = await Country.create({ name: 'Spain', flag: 'em-es' })
+        const Taiwan = await Country.create({
+            name: 'Taiwan',
+            flag: 'em-flag-tw',
+        })
+        const Thailand = await Country.create({
+            name: 'Thailand',
+            flag: '<em-flag-th',
```

Comment:
There is an extra "<" symbol here is that done on purpose?

Reply:
Already took it off,thank you!